mi/instrument/kut/ek60/ooicore/test/test_zplsc_b_echogram.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+from mi.instrument.kut.ek60.ooicore.zplsc_b import parse_datagram_file_wrapper
+
+parse_datagram_file_wrapper('/Users/admin/ooi-src/mi-instrument/mi/instrument/kut/ek60/ooicore/resource/'
```

Comment:
is hardcoding the path acceptable here?  can a relative path be used instead?

Reply:
I refactored the code to grab the raw data file from the raw data server and generate the file path for the local file based on the file path of the script.